projects/client-vscode/src/webview.ts

```diff
@@ -1,51 +1,76 @@
-import { ClientConnector, connectClient, applyApi, Client } from '@remixproject/plugin/connector'
-import { PluginClient } from '@remixproject/plugin/client'
-import { Message } from '../../utils/src/types/message'
-import { Api, ApiMap } from '../../utils/src/types/api'
-
+import {
+  ClientConnector,
+  connectClient,
+  applyApi,
+  Client,
+  PluginClient,
+  Message,
+  Api,
+  ApiMap,
+  isHandshake,
+  PluginOptions,
+  checkOrigin
+} from '@remixproject/plugin'
 
 declare const acquireVsCodeApi: any
-
 /**
  * This Webview connector
  */
 export class WebviewConnector implements ClientConnector {
```

Comment:
shouldn't this be in a `connector.ts` (like for iframe.ts) or is this fine to keep it here?

Reply:
I think `connector.ts` in iframe should be renamed. But I'm still figuring out how to organize this repo. I' happy to update, but I would prefer to se how it comes out.